docs/platforms/javascript/common/tracing/instrumentation/opentelemetry.mdx

```diff
@@ -113,3 +113,72 @@ tracer.startActiveSpan("span name", () => {
 ```
 
 You can also use any other tracer; all OpenTelemetry spans will be picked up by Sentry automatically.
+
+## Using a Custom Sampler
+
+We recommend using `SentrySampler` as this will correctly will ensure the correct subset of traces is sent to Sentry depending on your `tracesSampleRate`.
```

Comment:
```suggestion
We recommend using `SentrySampler` as this will ensure the correct subset of traces is sent to Sentry depending on your `tracesSampleRate`, as well as that all other Sentry features like trace propagation work as expected.
```